src/setup/setup.spec.ts

```
@@ -177,4 +177,80 @@
+    const recaptchaKeyQuestion = questionArray.find(
+      (q: { name?: string; validate?: (input: string) => boolean | string }) =>
+        q.name === 'recaptchaSiteKeyInput',
+    );
+
+    if (recaptchaKeyQuestion && recaptchaKeyQuestion.validate) {
+      capturedValidationFn = recaptchaKeyQuestion.validate;
+    }
+
+    // Test the validation function with valid input
+    expect(capturedValidationFn).toBeDefined();
+    const result = capturedValidationFn!('valid-key');
```

ESLint

[error] Forbidden non-null assertion.

(@typescript-eslint/no-non-null-assertion)